pilot/pkg/model/jwks_resolver.go

```diff
@@ -236,48 +239,33 @@ func (r *JwksResolver) GetPublicKey(issuer string, jwksURI string) (string, erro
 		}
 		return e.pubKey, nil
 	}
-
-	var err error
-	var pubKey string
-	if jwksURI == "" {
-		// Fetch the jwks URI if it is not hardcoded on config.
-		jwksURI, err = r.resolveJwksURIUsingOpenID(issuer)
-	}
-	if err != nil {
-		log.Errorf("Failed to jwks URI from %q: %v", issuer, err)
-	} else {
-		var resp []byte
-		resp, err = r.getRemoteContentWithRetry(jwksURI, networkFetchRetryCountOnMainFlow)
-		if err != nil {
-			log.Errorf("Failed to fetch public key from %q: %v", jwksURI, err)
-		}
-		pubKey = string(resp)
-	}
-
 	r.keyEntries.Store(key, jwtPubKeyEntry{
-		pubKey:            pubKey,
+		pubKey:            "",
```

Comment:
Please also update the comment of `GetPublicKey()`, the function name could also be updated to better reflect the new behavior.